b2/console_tool.py

```diff
@@ -1135,6 +1136,16 @@ class Sync(Command):
     Copies multiple files from source to destination.  Optionally
     deletes or hides destination files that the source does not have.
 
+    The synchronizer can copy files:
+
+    - From a B2 bucket to a local destination.
+    - From a local source to a B2 bucket.
+    - From one B2 bucket to another.
+    - Between different folders in the same B2 bucket.
+      It will sync only the latest versions of files.
```

Comment:
lets skip that line, because if we put it here, we'll have to increase major semver to release a `--mode=versions` feature